source/IO/dataset_import/VDJdbImport.py

```diff
@@ -134,19 +134,32 @@ def preprocess_dataframe(df: pd.DataFrame, params: DatasetImportParams):
                               f"To import all chains as a SequenceDataset, use paired = False")
         else:
             df.loc[df["sequence_identifiers"] == "0", "sequence_identifiers"] = None
-        # todo: should sequence identifiers be made unique?
 
         if "chains" not in df.columns:
             df.loc[:, "chains"] = ImportHelper.load_chains_from_genes(df)
 
+        df["receptor_identifiers"] = df["sequence_identifiers"]
```

Comment:
do we check that sequence identifiers that we assign to receptor identifiers are unique? (that there is one id per receptor) i can't seem to find it 🤔 👀

Reply:
we do not check it, but this is the point of VDJdb: complex.id (which is renamed to sequence_identifiers) is what VDJdb uses to tell different receptor pairs apart